README.md

```diff
@@ -82,6 +82,12 @@ There is one special external dependency, Boost:
 Then you just need to run `cargo run --release` (for a release build) or `cargo run` (for a debug build) to build and run touchHLE. On an underpowered, passively-cooled, 2-core laptop (2017 Retina MacBook), a clean release build takes a bit less than 9 minutes.
 
 The `touchHLE_dylibs` and `touchHLE_fonts` directories contain files that the resulting binary will need at runtime, so you'll need to copy them if you want to distribute the result. You also should include the license files.
+# Similar projects
+- [https://systems.cs.columbia.edu/projects/cycada/](Cycada), a kernel level iOS compatibility layer (iOS 5.1.1, with iOS 6 support partially) for Android worked on roughly from 2014 to 2017. It was never released due to closed source changes, however it can be reproduced via the papers with enough effort, see [http://jeremya.com/files/pub/2015/02/andrus-thesis.pdf](here) for the paper, we will pay you if you manage to create this in it's entirety.
```

Comment:
> we will pay you if you manage to create this in it's entirety.

Who will, and how much? The average user that sees this will assume that TouchHLE's developer themselves will pay them an unspecified sum of money for developing a public release of Cycada, and not an unknown third-party.

Reply:
Oh I didn't notice that bit, I'm going to close this just to ensure I don't merge it accidentally.

Reply:
FWIW I don't mind contributions that touch the README, but in general please be careful what words you're putting in my mouth here. I know it belongs to the project, but as a practical matter, I am the project for the time being.